examples/cloudrun/wait-approval/.pipe.yaml

```diff
@@ -0,0 +1,26 @@
+# Deployment pipeline that contains a manual approval stage.
+apiVersion: pipecd.dev/v1beta1
+kind: CloudRunApp
+spec:
+  notification:
+    pipeline:
+    stages:
+      # Promote new version to receive amount of traffic.
+      - name: CLOUDRUN_PROMOTE
+        with:
+          percent: 10
+      - name: WAIT_APPOVAL
+      # Promote new version to receive amount of traffic.
+      - name: CLOUDRUN_PROMOTE
+        with:
+          percent: 50
+      - name: WAIT_APPOVAL
```

Comment:
nit: To be simple, we can remove this part.

Reply:
769ebe3